okhttp/src/main/kotlin/okhttp3/internal/http/CallServerInterceptor.kt

```diff
@@ -20,6 +20,7 @@ import java.net.ProtocolException
 import okhttp3.Interceptor
 import okhttp3.Response
 import okhttp3.internal.EMPTY_RESPONSE
+import okhttp3.internal.http2.ConnectionShutdownException
 import okio.buffer
```

Comment:
the diffs on this file are worth viewing in 'ignore whitespace' mode because there’s a lot of indentation change